### PR TITLE
add: config & fix: delete entities & feat: return response

### PR DESCRIPTION
**Pull request by qiuhuiming**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Config:
+    @staticmethod
+    def get_connection_string():
+        return 'DefaultEndpointsProtocol=[http|https];AccountName=mylibrary030;AccountKey=k+HGhydwZIsniLJtVJA+j+4rormfbo5AuPaxcRB8gOqUMBSRIWyPE1MXdoEhEY5Qe7QReSqps3rjKM4UpcpN+w=='
```
Comment (hund030) [inline config.py]: The connection string is sensitive, should not appear in your PR.
Note: never submit your username, password, access key...

File: entity_manager.py [hunk 2]
```diff
@@ -30,10 +31,8 @@ def create_entity(client: TableClient, row_key: str) -> bool:
             return False
 
     @staticmethod
-    def query_entities(client: TableClient) -> list:
+    def query_entities(client: TableClient) -> ItemPaged:
```
Comment (hund030) [inline entity_manager.py:34]: Good to replace the type. The ItemPaged is a iterator, not a list.

File: main.py [hunk 3]
```diff
@@ -28,24 +30,33 @@ def delete_table(table_name: str):
...
+    return Response.ok()
 
 
 @app.get("/entities")
 def query_entities(table_name: str):
     client = ClientFactory.getTableClient(table_name)
     result = EntityManger.query_entities(client)
-
+    if not result:
+        raise HTTPException(status_code=500, detail='Cannot create the entity.')
```
Comment (hund030) [inline main.py]: Confused by the error message. I assume it should be something like `Cannot query entities`.

File: main.py [hunk 4]
```diff
@@ -28,24 +30,33 @@ def delete_table(table_name: str):
...
+    data = []
+    for item in result:
+        data.append(item)
```
Comment (hund030) [inline main.py]: We can replace for loop with iterator. Example:
```
data = [item for item in result]
```

File: main.py [hunk 5]
```diff
@@ -28,24 +30,33 @@ def delete_table(table_name: str):
...
         raise HTTPException(status_code=400, detail='Entity already exists.')
+    return Response.ok()
 
 
 @app.delete("/entities")
 def delete_entity(table_name: str, row_key: str, partition_key: str):
     client = ClientFactory.getTableClient(table_name)
     result = EntityManger.delete_entity(client, row_key, partition_key)
-    raise HTTPException(status_code=400, detail='Entity does not exist.')
+    if not result:
```
Comment (hund030) [inline main.py:58]: Thanks for bug fix.

File: response.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+        response = {
+            'code': 200,
+            'msg': 'success',
+        }
+        if data is not None:
+            response['data'] = data
+        return response
+    
+    @staticmethod
+    def error(code, msg):
```
Comment (hund030) [inline response.py:15]: This method is defined but never used. Maybe we can wrap HTTPException in it.
```
def error(code, msg):
    raise HTTPException(code, msg)
```

```
if not result:
    return Response.error(400, 'message')
```

File: main.py (hunk 3 above)
Comment (qiuhuiming) [reply main.py]: My careless about the detail. It should have been "Cannot query entities."

**Pull request merged**